plugins/inputs/t128_graphql/config.go

```diff
@@ -0,0 +1,76 @@
+package t128_graphql
+
+import (
+	"strings"
+)
+
+//Config stores paths and paths to fields, tags and predicates used by queryBuilder and responseProcessor
+type Config struct {
+	QueryPath  string
+	Predicates map[string]string
+	Fields     map[string]string
+	Tags       map[string]string
+	RawFields  map[string]string //Fields is used with RawFields to build the query
+	RawTags    map[string]string
+}
+
+//LoadConfig converts an entry point into a corresponding responsePath, queryPath and predicates
+func LoadConfig(entryPoint string, fieldsIn map[string]string, tagsIn map[string]string) *Config {
```

Comment:
@salegria8 can you throw in an example of what the entryPoint string looks like. It will help to understand the parsing logic